tests/test_env_string.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pytest
+
+from vcr_cleaner.cleaners.env_strings import clean_env_strings
+
+
+def get_interaction(body):
+    return {
+        'response': {
+            'body': {
+                'string':body
+            }
+        }
+    }
+
+
+@pytest.mark.parametrize('body,expected', [
+    ('CLEAN ME', 'CLEANED'),
+    ('CLEAN THIS TOO', 'CLEANED'),
+    ('CLEAN ME and CLEAN THIS TOO', 'CLEANED and CLEANED'),
+    ('NOTHING TO CLEAN', 'NOTHING TO CLEAN'),
+    (b'DEADBEEF', b'DEADBEEF'),
```

Comment:
You may want to add another test to make it clear that binary strings are not cleaned:

```suggestion
    (b'DEADBEEF', b'DEADBEEF'),
    (b'CLEAN ME', CLEAN ME'),  # Binary strings are not cleaned
```

Reply:
Great point. Done.